### DISCUSSION TITLE
Will Windows 10 version 21H2 API set be based on Windows Server 2022?

### DISCUSSION THREAD

**Discussion by jaigak** [microsoft/WindowsAppSDK#1319] [General] [answered]
Windows Server 2022's WinRT API set has a few things that I would like to see on Windows 10. Will those be available on Windows 10 version 21H2?

**Comment by btueffers** [comment, marked as answer]
Hi @Jaiganeshkumaran, thank you for starting this discussion! It's not related to Windows App SDK so a better forum for this conversation would be [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/topics/windows-uwp.html).

**Reply by jaigak** [reply]
https://docs.microsoft.com/en-us/answers/questions/548172/will-windows-10-version-21h2-api-set-be-based-on-w.html
